### PR TITLE
3djc/taranis fix pots checks

### PR DESCRIPTION
**Pull request by 3djc**
WARNING :
-you WILL need to go in hardware menu to check sliders assignement
-you also NEED to go to calibration (where you will now see the 4 sliders on X9E)

### CODE REVIEW

File: radio/src/gui/212x64/radio_hardware.cpp [hunk 1]
```diff
@@ -115,14 +113,16 @@ void menuRadioHardware(event_t event)
...
       case ITEM_RADIO_HARDWARE_LS2:
       case ITEM_RADIO_HARDWARE_RS2:
       {
-        int idx = k - ITEM_RADIO_HARDWARE_LS2;
+        int idx = k - ITEM_RADIO_HARDWARE_LS;
         uint8_t mask = (0x01 << idx);
-        lcdDrawTextAtIndex(INDENT_WIDTH, y, STR_VSRCRAW, NUM_STICKS+NUM_XPOTS+2+idx+1, menuHorizontalPosition < 0 ? attr : 0);
-        if (ZEXIST(g_eeGeneral.anaNames[NUM_STICKS+NUM_XPOTS+2+idx]) || (attr && menuHorizontalPosition == 0))
-          editName(HW_SETTINGS_COLUMN, y, g_eeGeneral.anaNames[NUM_STICKS+NUM_XPOTS+2+idx], LEN_ANA_NAME, event, attr && menuHorizontalPosition == 0);
+        lcdDrawTextAtIndex(INDENT_WIDTH, y, STR_VSRCRAW, NUM_STICKS+NUM_XPOTS+idx+1, menuHorizontalPosition < 0 ? attr : 0);
```
Comment (bsongis) [inline radio/src/gui/212x64/radio_hardware.cpp]: NUM_POTS instead of NUM_XPOTS, no?

File: radio/src/gui/212x64/radio_hardware.cpp [hunk 2]
```diff
@@ -115,14 +113,16 @@ void menuRadioHardware(event_t event)
...
+        if (ZEXIST(g_eeGeneral.anaNames[NUM_STICKS+NUM_XPOTS+idx]) || (attr && menuHorizontalPosition == 0))
+          editName(HW_SETTINGS_COLUMN, y, g_eeGeneral.anaNames[NUM_STICKS+NUM_XPOTS+idx], LEN_ANA_NAME, event, attr && menuHorizontalPosition == 0);
```
Comment (bsongis) [inline radio/src/gui/212x64/radio_hardware.cpp]: NUM_POTS here as well

File: radio/src/gui/212x64/radio_hardware.cpp [hunk 3]
```diff
@@ -115,14 +113,16 @@ void menuRadioHardware(event_t event)
         break;
       }
 #if defined(PCBX9E)
+      case ITEM_RADIO_HARDWARE_LS:
```
Comment (bsongis) [inline radio/src/gui/212x64/radio_hardware.cpp]: But then the name of LS and RS is not anymore configurable on X9D / X9D+
I suggest only the 2 case lines for LS2 and RS2 are surrounded by the ```#if defined```

**Comment by 3djc** [comment]
There is a remaining bug

**Pull request closed without merging**